data/data/bootstrap/baremetal/README.md

```diff
@@ -22,19 +22,33 @@ Relevant files:
 * **systemd/units/keepalived.service** - systemd unit file for `keepalived`.
   This runs `keepalived.sh` to generate the proper configuration from the
   template and then runs podman to launch `keepalived`.
+* **files/usr/local/bin/fletcher8** - 
+* **files/usr/local/bin/get_vip_subnet_cidr** - 
 
 ## Internal DNS
 
 The bootstrap assets relating to DNS automate as much of the DNS requirements
 internal to the cluster as possible.
 
-TODO - explain how this works ...
+There is a DNS VIP managed by `keepalived` in a manner similar to the API VIP
+discussed above.
+
+`coredns` runs with a custom `mdns` plugin (`coredns-mdns`) that can
+dynamically generate the DNS SRV record for `etcd`, as well as resolve the
+`etcd` hostnames.
 
 Relevant files:
-* files/etc/coredns/Corefile
-* files/etc/keepalived/keepalived.conf.tmpl
-* files/etc/dhcp/dhclient.conf
-* files/usr/local/bin/fletcher8
-* files/usr/local/bin/get_vip_subnet_cidr
-* files/usr/local/bin/coredns.sh
-* systemd/units/coredns.service
+* **files/etc/keepalived/keepalived.conf.tmpl** - `keepalived` configuration
+  template, includes configuration for the DNS VIP
```

Comment:
Should we add a note here about the filename extension, e.g it's not .template because the templating is done via envsubst in https://github.com/openshift-metal3/kni-installer/blob/master/data/data/bootstrap/baremetal/files/usr/local/bin/keepalived.sh and not via the typical go templating inside the installer?

Reply:
added a note